ui/src/components/DashboardInfoModal/DashboardInfoModal.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import Modal from '@splunk/react-ui/Modal';
+import Message from '@splunk/react-ui/Message';
+import styled from 'styled-components';
+import Heading from '@splunk/react-ui/Heading';
+import P from '@splunk/react-ui/Paragraph';
+import QuestionCircle from '@splunk/react-icons/QuestionCircle';
+
+import { StyledButton } from '../../pages/EntryPageStyle';
+
+const ModalWrapper = styled(Modal)`
+    width: 700px;
+`;
+
+interface DashboardInfoModalProps {
+    open: boolean;
+    title: string;
+    subtitle?: string;
+    handleRequestClose: () => void;
+    errorTypesInfo: { header: string; description: string }[];
+    closeBtnLabel?: string;
+    infoMessage?: string;
+    listIntroductionText?: string;
+    troubleshootingButton?: { label?: string; link?: string };
+}
+
+function DashboardInfoModal(props: DashboardInfoModalProps) {
```

Comment:
the component is too generic so the value is diminished, but the boilerplate is huge.
The amount of props is kind of overwhelming, do we need all of these or we are going to reuse this component with different params?

Reply:
yea actually i did not refactor that one after creating as in the end it is currently not used, but maybe worth changing it even now to not use it after without changes.
i will fix it

Reply:
actually since we got it all ready, approved and that component is actually currently not used i will fix in separate PR, just to close that one